### DISCUSSION TITLE
Help for Twitter bot (twitter api, caarlos0/twitter-cleaner)

### DISCUSSION THREAD

**Discussion by lalapoto0011** [community/community#53890] [Programming Help]
### 

Hi, I'm trying to clear a tweet from my account using the code on [this page. (caarlos0/twitter-cleaner)](https://github.com/caarlos0/twitter-cleaner/tree/v0.7.0)
When running on Windows via `go run main.go`, this error message appears.

```
go run main.go
  • deleting tweets from twitter timeline
  ⨯ failed to delete tweets from timeline            error=failed to clean up timeline: failed to get timeline: Get https://api.twitter.com/1.1/statuses/user_timeline.json?count=200&include_rts=true&tweet_mode=extended returned status 403, {"errors":[{"message":"You currently have Essential access which includes access to Twitter API v2 endpoints only. If you need access to this endpoint, you’ll need to apply for Elevated access via the Developer Portal. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve","code":453}]}

exit status 1
```

Can you tell me why this error is coming out?
Is it because of the reorganization of Twitter api?
Or is it because I don't have authority?
I'm using Free on Twitter API v2.
Before the reorganization, it was 'Essential'.
(The Twitter API has changed now, so I can't apply for an 'Elevated' account.)

Any help would be appreciated!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mitra-tabatabaei** [comment]
i need to Forbidden: 403 Forbidden
453 - You currently have access to a subset of Twitter API v2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.twitter.com/en/portal/product

**Comment by RZ7272** [comment]
> Hi, I'm trying to clear a tweet from my account using the code on [this page. (caarlos0/twitter-cleaner)](https://github.com/caarlos0/twitter-cleaner/tree/v0.7.0) When running on Windows via `go run main.go`, this error message appears.
> 
> ```
> go run main.go
>   • deleting tweets from twitter timeline
>   ⨯ failed to delete tweets from timeline            error=failed to clean up timeline: failed to get timeline: Get https://api.twitter.com/1.1/statuses/user_timeline.json?count=200&include_rts=true&tweet_mode=extended returned status 403, {"errors":[{"message":"You currently have Essential access which includes access to Twitter API v2 endpoints only. If you need access to this endpoint, you’ll need to apply for Elevated access via the Developer Portal. You can learn more here: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve","code":453}]}
> 
> exit status 1
> ```
> 
> Can you tell me why this error is coming out? Is it because of the reorganization of Twitter api? Or is it because I don't have authority? I'm using Free on Twitter API v2. Before the reorganization, it was 'Essential'. (The Twitter API has changed now, so I can't apply for an 'Elevated' account.)
> 
> Any help would be appreciated!
> 
> ### Guidelines
> * [x]  I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.
